400-claims.md

```diff
@@ -128,64 +128,105 @@ The parameter data stored in a claim data is _the resolved key/value pairs_ that
   - Valid user-supplied values are presented
   - Default values are supplied for all parameters where `default` is provided and no user-supplied value overrides this
 
-### Outputs
+### Claim Results
+
+The result of executing an operation is defined separately in a claim result document. A claim may have multiple results. For example if the operation was cancelled and then retried the claim will have two results: one representing the cancelled operation and another the retried operation.
 
-The claim `result` provides a list of output names generated by the operation. A tool may choose to request the contents of any outputs to persist them but is not required to do so.
+```json
+{
+  "claimRef": {
+    "installation": "technosophos.helloworld",
+    "revision": "01CP6XM0KVB9V1BQDZ9NK8VP29"
+  },
+  "custom": {},
+  "id": "01E2ZZ2FKSE0V41DCXFCSW5D1M",
+  "created": "2018-08-30T20:39:55.549002887-06:00",
+  "modified": "2018-08-30T20:39:59.611068556-06:00",
+  "message": "",
+  "outputs": {
+    "clientCert": {},
+    "hostName": {},
+    "port": {}
+  },
+  "started": "2018-08-30T20:39:55.549002887-06:00",
+  "status": "succeeded",
+  "stopped": "2018-08-30T20:39:59.611068556-06:00"
+}
+```
+Source: [400.01-claim-result.json](examples/400.01-claim-result.json)
+
+The fields above are defined as follows:
+
+- `claimRef` (REQUIRED): Reference to the claim that generated this result.
+  - `installation` (REQUIRED): The claim installation name.
+  - `revision` (REQUIRED): The claim revision.
+- `custom` (OPTIONAL): A section for custom extension data applicable to a given runtime.
+- `id` (REQUIRED): An [ULID](https://github.com/ulid/spec).
+- `created` (REQUIRED): A timestamp indicating when this result was first created. This MUST not be changed after initial creation.
+- `modified` (REQUIRED): A timestamp indicating the last time this result was modified. Creating the result MUST set this to a timestamp that matches the result's `created`.
+- `message` (OPTIONAL): A human-readable string that communicates the outcome. Error messages MAY be included in `failed` conditions.
+- `outputs` (OPTIONAL): Map of output names generated by the operation. These are stored so that the user can access them after the operation completes. Some implementations MAY choose not to store these for security or portability reasons. See the [Outputs](#outputs) section for details. If this field is not present, it may be assumed that no outputs were generated as a result of the operation.
```

Comment:
Is it worth mentioning that output values MAY be included as well, since the object itself is still a key-value map?  Or, is there agreement that actual output values should not be included (values always empty)?

Reply:
Oh, yeah let me clarify that output values are NOT in the claim result. The struct is reserved for future use (coming in another PR) containing metadata about the output. We want to put stuff in there like the content digest for example. But the full output value wouldn't be in there anymore.